assembl/models/preferences.py

```diff
@@ -289,8 +290,12 @@ def validate_single_value(self, key, value, pref_data, data_type):
                     "value not allowed: " + value)
             elif data_type == "url":
                 from urlparse import urlparse
-                assert urlparse(value).scheme in (
-                    'http', 'https'), "Not a HTTP URL"
+                result = urlparse(value).scheme in (
+                    'http', 'https')
+                if not result:
+                    # Check if it's an IP address
+                    result = is_valid_ipv4_address(value)
+                assert result, "Not a valid address"
```

Comment:
maybe you could keep the old `assert` to have a different error message if it's not a http url?

Reply:
Sure, can look further into it.